analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -571,6 +571,7 @@ class MavenSupport(workspaceReader: WorkspaceReader) {
         }
 
         val browsableScmUrl = getOriginalScm(mavenProject)?.url
+        val isBomArtifact = mavenProject.packaging == "pom"
```

Comment:
I'm always a bit hesitant to use the term "BOM" in the Maven context, because [not every artifact with `pom` packaging is a BOM](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#bill-of-materials-bom-poms): It could also be a [parent or aggregation](https://maven.apache.org/pom.html#inheritance) POM. 

That's why I'd prefer this to be simply called `isPomPackaging`, or even just inline the variable.

Reply:
Done. Updated the commit message accordingly.